internal/scan-manager/webapp/src/main/java/org/wso2/security/tools/scanmanager/webapp/controller/ScanController.java

```diff
@@ -118,8 +120,15 @@ public String startScan(MultipartHttpServletRequest multipartHttpServletRequest)
                 parameterMap.put(requestParamMap.getKey(), requestParamMap.getValue()[0]);
             }
         }
-        String username = (String) (((StandardMultipartHttpServletRequest) multipartHttpServletRequest).getRequest())
-                .getSession().getAttribute(SCAN_REQUEST_USER_NAME);
+        String username = null;
+        Authentication authentication = SecurityContextHolder.getContext()
```

Comment:
Can `authentication` become null?

Reply:
No. If that value is null, scan manager redirects it to Identity Provider to login. Therefore when it comes to this line, always authentication has a value.